### DISCUSSION TITLE
Upper right corner cards(?)

### DISCUSSION THREAD

**Discussion by DanMumford** [qnblackcat/uYouPlus#659] [Q&A]
Is it possible for uYou to show those upper corner cards you get on certain videos (with half a password etc?)

![399E7928-1797-465E-977A-9EA7AC6B0FF0](https://user-images.githubusercontent.com/73539174/200354679-e0487689-6e5d-4e57-8391-533320505a5e.jpeg)

Thanks! 👍

**Comment by qnblackcat** [comment]
No, sorry
